Update eligibility for veterans bursary (#2789)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/if-youre-a-veteran.md

## Undergraduate bursary for veterans

> You may be eligible for a tax-free undergraduate bursary of £40,000 if you (all of the following):

Reviewer: ```suggestion
You may be eligible for a tax-free undergraduate bursary of £40,000 if you are (all of the following):
```
Feels a bit funny like it doesn't read write the 'if you (all of the following)'. Could we amend slightly?

> * have left full-time employment with the British Army, Royal Air Force or Royal Navy no more than 5 years before the start of the course

Reviewer: ```suggestion
* a veteran who has left full-time employment with the British Army, Royal Air Force or Royal Navy no more than 5 years before the start of the course
```

> * are training to teach secondary biology, physics, chemistry, computing, maths or languages

Reviewer: ```suggestion
* training to teach secondary biology, physics, chemistry, computing, maths or languages
```

> * are doing an undergraduate degree leading to [qualified teacher status (QTS)](/what-is-qts) in England

Reviewer: ```suggestion
* doing an undergraduate degree leading to [qualified teacher status (QTS)](/what-is-qts) in England
```
